### PR TITLE
Remove listings and events

### PR DESCRIPTION
**Pull request by ticruz38**
Removing calendar events as well as the @event-calendar dependency.

EventDetail, EventEdit and EventDelete are still accessible via an Event Note so they remain.

#452
#453

### CODE REVIEW

File: src/app/views/NoteCreate.svelte [hunk 1]
```diff
@@ -126,18 +125,6 @@
...
-      calendar_event: () =>
-        createEvent(31923, {
-          content,
-          tags: [
-            ...tags,
-            ["d", uuid()],
-            ["title", opts.title],
-            ["location", opts.location || ""],
-            ["start", dateToSeconds(opts.start).toString()],
-            ["end", dateToSeconds(opts.end).toString()],
```
Comment (staab) [inline src/app/views/NoteCreate.svelte:138]: These can be removed from other places in the file.

**Review by staab — COMMENTED** [review]
Go ahead and remove the EventActions stuff, and the delete/edit links from EventInfo. That will allow us to remove even more. And since users can't create events in coracle any more, it doesn't make sense to let them edit/delete.

**Comment by ticruz38** [comment]
- removed all things related to listing and calendar in NoteCreate
- remove EventDetail and EventDelete and all actions/buttons/links referring it

**Comment by staab** [comment]
You should be able to completely remove EventActions and EventDetail. Be sure to remove the `detailPath` stuff in `EventInfo` as well.

**Comment by ticruz38** [comment]
- Removed EventDetail too
- Kept EventAction only for the QR code sharing, if not, I'll remove EventActions completely

**Comment by staab** [comment]
Yeah, go ahead and remove EventActions completely. We can add qr code sharing to normal event share if we decide we want that down the line.

**Pull request merged**